content/id/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1062 @@
+---
+title: Secret
+content_template: templates/concept
+feature:
+  title: Secret dan manajemen konfigurasi
+  description: >
+    Menerapkan serta mengubah secret serta konfigurasi aplikasi tanpa melakukan perubahan pada image kamu serta mencegah tereksposnya secret yang kamu miliki pada konfigurasi.
+weight: 50
+---
+
+
+{{% capture overview %}}
+
+Objek `secret` pada Kubernetes mengizinkan kamu menyimpan dan mengatur informasi yang sifatnya sensitif, seperti
+_password_, token OAuth, dan ssh _keys_. Menyimpan informasi yang sifatnya sensitif ini ke dalam `secret`
+cenderung lebih aman dan fleksible jika dibandingkan dengan menyimpan informasi tersebut secara apa adanya pada
+definisi {{< glossary_tooltip term_id="pod" >}} atau di dalam {{< glossary_tooltip text="container image" term_id="image" >}}.
```

Comment:
Some lines are still breaking for the glossary, not sure why
![image](https://user-images.githubusercontent.com/3192347/61197586-f2aff580-a6ff-11e9-958c-96942dfb8cba.png)

any clue @raelga ?

Reply:
https://deploy-preview-14792--kubernetes-io-master-staging.netlify.com/docs/concepts/configuration/secret/

Reply:
Looking into it! 🔍

Reply:
I already tried to use editorconfig, but still nothing happened 😞 
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
https://www.jetbrains.com/help/idea/configuring-code-style.html